test/validation/api/ipsec/ipsec_test_out.c

```diff
@@ -471,7 +471,7 @@ static void test_out_in_common(ipsec_test_flags *flags,
 			  .status.error.all = 0,
 			  .l3_type = ODP_PROTO_L3_TYPE_IPV4,
 			  .l4_type = ODP_PROTO_L4_TYPE_ICMPV4,
```

Comment:
Yes it is. It is fixed later in this patch:

472a9dd8c validation: ipsec: always check packet type after ipsec output

That patch fixes also other similar problems and adds actual checking of the l3/l4 types so that the values start to matter (which is why I did not do the fix yet in this patch).